tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/documentation/DeprecatedAnnotationTest.java

```diff
@@ -310,6 +310,29 @@ public void testDeprecatedObjectMemberField() {
         }
     }
 
+    @Test(description = "Test @deprecated annotation for record field")
+    public void testDeprecatedRecordField() {
+        List<Record> records = testModule.records;
+        Record employeeRecord = null;
+
+        for (Record record : records) {
+            String reocrdName = record.name;
+            if ("Employee".equals(reocrdName)) {
```

Comment:
Typo. But we don't need a separate variable anyway?

```suggestion
            if ("Employee".equals(record.name)) {
```